config/v201/init_core.sql

```diff
@@ -17,6 +17,18 @@ CREATE TABLE IF NOT EXISTS AVAILABILITY(
     CHECK (EVSE_ID >= 0 AND CONNECTOR_ID >= 0 AND (EVSE_ID > 0 OR CONNECTOR_ID = 0))
 );
 
+-- Transactions --
+CREATE TABLE IF NOT EXISTS TRANSACTIONS (
+    TRANSACTION_ID TEXT PRIMARY KEY NOT NULL,
+    SEQ_NO INT NOT NULL,
+    MESSAGE_TYPE TEXT NOT NULL,
```

Comment:
`MESSAGE_TYPE` is not used and I expected the start meter value to be in here for recovery.